### ISSUE TITLE
Ignore folders and archives are missing

### ISSUE THREAD

**Issue by nerdseverino** [eldertek/duplicatefinder#4]
After upgrade to 1.0.2 config to ignore folders and archives are missing 
anyone can help me?
Nextcloud 27 - Docker php-fpm image.

**Comment by eldertek** [comment]
Hello, as you can see in changelogs.

There is a regression of these functions from webui, you can edit directly config with occ.

I am working for the coming back of the functionality.

**Comment by eldertek** [comment]
On /10, what is you need ?

**Comment by asaage** [comment]
In the https://github.com/nextcloud/fulltextsearch/ you can ignore folder-indexing if you put a ```.noindex``` file into the folder you'd like to exclude. Same goes for the photos - app which ignores folders with a ```.nomedia``` file.
I'd like to see that kind of functionality here.
Either respecting a ```.noindex``` file or using some other file like ```.nodupefinder```.

**Comment by eldertek** [comment]
> In the https://github.com/nextcloud/fulltextsearch/ you can ignore folder-indexing if you put a `.noindex` file into the folder you'd like to exclude. Same goes for the photos - app which ignores folders with a `.nomedia` file. I'd like to see that kind of functionality here. Either respecting a `.noindex` file or using some other file like `.nodupefinder`.

That’s a good idea, I’ll take a look. However, pull requests are not unwelcome ! =)

**Comment by eldertek** [comment]
@asaage 

Thank you for your suggestion! We always appreciate feedback and new ideas. The release incorporating your enhancement is now available on the App Store. For more details on the functionality, please refer to our updated README.

Closing as completed by  [a6e00fac61c4609ea59b1c2e948aa1cb6c9b675c](https://github.com/eldertek/duplicatefinder/commit/a6e00fac61c4609ea59b1c2e948aa1cb6c9b675c)

**Comment by asaage** [comment]
That was approximately months quicker than i would ever have been able to 😉

**Comment by eldertek** [comment]
That took you months less than it would've taken me! 😄 Open to all suggestions, and translations are welcome too! no only code pull request ahah😂

**Comment by asaage** [comment]
Is that other feature scrapped? (i never saw that)
That would have allowed to ignore in a more controllable way (like for example ignore certain filetypes) and without actualy putting hidden files into the directorystructure i guess.

**Comment by eldertek** [comment]
> Is that other feature scrapped? (i never saw that) That would have allowed to ignore in a more controllable way (like for example ignore certain filetypes) and without actualy putting hidden files into the directorystructure i guess.

Yes, I had to remove it since it was too complex for me to maintain. But I believe in the future, with more resources, I'll be able to introduce a more refined conditional detection based on file type, name, hash, etc.

**Comment by asaage** [comment]
understandably so

> introduce a more refined conditional detection based on file type, name, hash, etc.

That might be nesseccary as you can't always put aditional files in places.
Keep up the good work...

**Comment by eldertek** [comment]
Thank you, I will create a new issue then add to milestone and will work on when I Will have free time.

**Comment by jochen-01** [comment]
Re. .nodupefinder
What about something like .nodupetree to exclude a whole sub tree from duplicate search?
